code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -31,7 +31,7 @@
 		/obj/item/stock_parts/manipulator = 2,
 		/obj/item/stack/sheet/glass = 1)
 
-/obj/item/circuitboard/machine/abductor
+//obj/item/circuitboard/machine/protolathe/abductor
 	name = "alien board (Report This)"
 	icon_state = "abductor_mod"
```

Comment:
You aren't even commenting this out right

Reply:
That's the git diff being really fucky. Check the actual file.